airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1362 @@
+    """
+    Hook for Google Cloud Data Loss Prevention (DLP) APIs.
+    Cloud DLP allows clients to detect the presence of Personally Identifiable
+    Information (PII) and other privacy-sensitive data in user-supplied,
+    unstructured data streams, like text blocks or images. The service also
+    includes methods for sensitive data redaction and scheduling of data scans
+    on Google Cloud Platform based data sets.
+
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :type gcp_conn_id: str
+    :param delegate_to: The account to impersonate, if any.
+        For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    def __init__(self,
+                 gcp_conn_id="google_cloud_default",
+                 delegate_to=None):
+        super().__init__(gcp_conn_id, delegate_to)
+        self._client = None
+
+    def get_conn(self):
+        """
+        Provides a client for interacting with the Cloud DLP API.
+
+        :return: GCP Cloud DLP API Client
+        :rtype: google.cloud.dlp_v2.DlpServiceClient
+        """
+        if not self._client:
+            self._client = DlpServiceClient(credentials=self._get_credentials())
+        return self._client
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def cancel_dlp_job(
+        self, name, retry=None, timeout=None, metadata=None
+    ):
+        """
+        Starts asynchronous cancellation on a long-running DlpJob.
+
+        :param name: The name of the DlpJob resource to be cancelled.
+        :type name: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        """
+
+        client = self.get_conn()
+
+        if not name:
+            raise AirflowException(
+                "Please provide the name of the DlpJob resource to be cancelled."
+            )
+
+        client.cancel_dlp_job(
+            name=name, retry=retry, timeout=timeout, metadata=metadata
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def create_deidentify_template(
+        self,
+        parent,
+        deidentify_template=None,
+        template_id=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        Creates a DeidentifyTemplate for re-using frequently used configuration for
+        de-identifying content, images, and storage.
+
+        :param parent: The parent resource name.
```

Comment:
```suggestion
        :param organization: The parent resource name.
```
Other operators take path arguments instead of paths, because this allows better integration with the project ID.
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/gcp_vision_hook.py#L154